.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test
+# https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/2
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+jobs:
+  test-vers:
+    strategy:
+      matrix:
+        node: ['8.6.0', '8', '10', '12', '14']
```

Comment:
Also -- just noting that we've dropped the Node 6 support that was in the old travis file -- this is appropriate.

Reply:
I guess. A drop of a node major version should, IMHO, be a major version bump. I'll re-add node 6 here for testing.